.travis.yml

```diff
@@ -55,6 +66,26 @@ before_install:
   - rm -rf tmp-build
   - if [[ -n $BUILD_TYPE ]]; then bash mk/bootstrap-librdkafka.sh ${LIBRDKAFKA_VERSION} tmp-build ; fi
   - go get -u golang.org/x/lint/golint && touch .do_lint
+  - mkdir -p "$GOPATH/src/gopkg.in/confluentinc/confluent-kafka-go.v1/"
+  - cp -r . "$GOPATH/src/gopkg.in/confluentinc/confluent-kafka-go.v1/"
```

Comment:
We're trying to move away from gopkg.in (in favour of go modules), do we really need this here?

Reply:
It looks like all gopkg.in references for confluent-kafka-go have been removed from the code. I will remove this